source/snooty.txt

```diff
@@ -65,6 +65,12 @@ is loaded into Visual Studio Code with a ``snooty.toml`` file in the root.
    .. code-block:: sh
 
       git clone https://github.com/mongodb/docs-compass.git
+
+   Then open the ``docs-compass`` project either through the Visual Studio Code
+   UI or with the following command:
+
+   .. code-block:: sh
+
```

Comment:
Comment for line 169 (sorry, github doesn't let you comment on non-diff lines 😡 ): what's a workspace in this context? Does it mean I can only have docs-compass open, but not docs-compass AND another docs-whatever that's snooty-enabled?

Reply:
Currently yeah, that's a limitation of snooty. I'll add a note to that effect!